spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    before(:example) { get user_posts_path(1) }
+
+    it 'should return a successful response' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'shoud render posts/index.html.erb template' do
+      expect(response).to render_template('index')
+    end
+
+    it 'response body should contain' do
+      expect(response.body).to include('Posts Page')
+    end
+  end
+
+  describe 'GET /show' do
+    before(:example) { get user_posts_path(1, 1) }
+
+    it 'should return a successful response' do
```

Comment:
As requested by the previous code reviewer your tests are failing it would be nice if you corrected this issue
![Screenshot 2023-02-10 at 10 20 32 AM](https://user
![Screenshot 2023-02-15 at 5 38 49 PM](https://user-images.githubusercontent.com/79506105/219077756-04140ba2-f372-4e48-994b-c4c777aaa55a.png)
-images.githubusercontent.com/79506105/219076907-9f6dbed1-4c62-4341-a36b-3ee872582dd9.png)

Reply:
Hi @KayLemba ,
Kindly can you give me a thorough review I doubled check it all my spec files and it seems all good, I think  the error I get after running  rspec it appears that be related to db and I still can't figure out the issue